### PR TITLE
Feature/hero image

### PR DESCRIPTION
**Pull request by stuaxo**
Add hero image + header text.

### CODE REVIEW

File: industry/templates/industry/blocks/heading.html [hunk 1]
```diff
@@ -1,4 +1,20 @@
+{% load wagtailmarkdown %}
+{% load wagtailimages_tags %}
```
Comment (richtier) [inline industry/templates/industry/blocks/heading.html]:
```
{% load image from wagtailimages_tags %}```

would be more explicit

File: industry/templates/industry/blocks/heading.html [hunk 2]
```diff
@@ -1,4 +1,20 @@
+{% load wagtailmarkdown %}
```
Comment (richtier) [inline industry/templates/industry/blocks/heading.html]: is this in use?

File: industry/templates/industry/blocks/heading.html (hunk 1 above)
Comment (stuaxo) [reply industry/templates/industry/blocks/heading.html]: Good catch, I've cargo culted that.

File: industry/templates/industry/blocks/heading.html (hunk 2 above)
Comment (stuaxo) [reply industry/templates/industry/blocks/heading.html]: Good catch.

**Pull request merged**